### DISCUSSION TITLE
TIL - 22/03/18

### DISCUSSION THREAD

**Discussion by DongGeon0908** [selab-hs/selab#135] [Check TIL]
@DongGeon0908 
@leeheefull

**Comment by DongGeon0908** [comment]
### 작업사항
- Auth 리펙토링 진행중
- Member 설계 작업 변경

### 이슈
- Member 엔티티 변경시 부가적으로 영향을 받는 클래스 탐색 필요

**Comment by leeheefull** [comment]
### 작업 사항
* ["슬라이스 테스트를 하게 된 이유"](https://github.com/selab-hs/selab-blog-api/wiki/%5B%ED%9D%AC%EC%B0%AC%5D-%EC%8A%AC%EB%9D%BC%EC%9D%B4%EC%8A%A4-%ED%85%8C%EC%8A%A4%ED%8A%B8%EB%A5%BC-%ED%95%98%EA%B2%8C-%EB%90%9C-%EC%9D%B4%EC%9C%A0) 관련하여 정리하고 wiki에 작성
* ["Fixture monkey"](https://github.com/selab-hs/selab-blog-api/wiki/%5B%ED%9D%AC%EC%B0%AC%5D-Fixture-monkey-%EB%8F%84%EC%9E%85) 관련하여 정리하고 wiki에 작성

### 이슈
